File: Python Word Examples/ConvertApiDemo/convertToPdf.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.convert_api.ConvertApi(configuration)

private_fonts = [PrivateFont("黑体", "input/simhei.ttf")]
extra = {'folder': 'input',
         'private_fonts': private_fonts,
         'dest_file_path': 'output/ConvertToPdf.pdf'}
api.convert_to_pdf('Template.docx', **extra)

File: PHP Word Examples/ConvertApiDemo/convertToPdf.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\ConvertApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new ConvertApi($configuration);

$name = "convert.docx";
$folder = "input";
$storage = null;
$password = null;
$dest_file_path = "output/ConvertToPdf.pdf";
$private_fonts = array(
    new PrivateFont(array(
        'name' => '黑体',
        'path' =>"DocDocument/simhei.ttf",
        'embed'=>true
    )),
    new PrivateFont(array(
        'name' => 'consola',
        'path' =>"DocDocument/consola.ttf",
        'embed'=>true
    ))
);
$apiInstance->convertToPdf($name, $dest_file_path, $password, $folder, $storage,$private_fonts);